2_process.R

```diff
@@ -61,5 +63,11 @@ p2_targets_list <- list(
         purrr::map(.,summarize_paired_comids) %>%
         bind_rows()
     }
-  )
+  ),
+  
+  tar_target(p2_LC_per_catchment, 
+             raster_to_catchment_polygons(polygon_sf = p1_catchments_sf,
+                                          raster = p1_backcasted_LC[1], categorical_raster = TRUE,
```

Comment:
Is this for only year 2000? `p1_backcasted_LC[1]`

Reply:
Or is this target missing a `map` over the rasters?

Reply:
yes only for year 1960 . Mapping not included yet. I plan to adjust this target to loop through all the files in `p1_backcasted_LC`.